### ISSUE TITLE
Fails on Mac

### ISSUE THREAD

**Issue by Benjamin-Lee** [AliYoussef96/BCAW-Tool#12]
I think it has to do with the fact that you're checking if the user is on Windows or Linux, which results in the unbound variable when on Mac.

```python
~ ❯❯❯ python test.py                                                                                                                    (bcaw)
Traceback (most recent call last):
  File "test.py", line 2, in <module>
    BCAWT.BCAW(['Ecoli.fasta'],'result_folder',genetic_code_=11,Auto=True)
  File "/usr/local/Caskroom/miniconda/base/envs/bcaw/lib/python3.7/site-packages/BCAWT/BCAWT.py", line 106, in BCAW
    if os.path.exists(directory) == False:
UnboundLocalError: local variable 'directory' referenced before assignment
~ ❯❯❯ ipython                                                                                                                           (bcaw)
Python 3.7.3 (default, Apr 13 2019, 18:43:32)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.6.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import platform

In [2]: platform.system()
Out[2]: 'Darwin'
```

**Comment by AliYoussef96** [comment]
I replaced one parameter in the main function, I believe this will solve the issue and will be compatible with any platform.

The new parameter (save_path) will take an absolute path to a directory to save the result in, with default value equal to a current directory.

https://github.com/AliYoussef96/BCAW-Tool/blob/9c03019bfcdeeeb783aa3f2336c163ec30e0c42d/BCAWT/BCAWT.py#L1

I did test the code on Windows10 and Ubuntu19 and it works fine. I don't have access to a Mac right now but with the current changes, I think It will work fine with you.

I updated the documentation, README, as well as the pip version.
**pip version now is 1.0.1.**
